crates/swc_ecma_minifier/tests/terser/compress/template_string/special_chars_in_string/output.js

```diff
@@ -1,3 +1,3 @@
-var str = "foo `;\n`${any} bar";
+var str = "foo `;\n\\`${any} bar";
```

Comment:
Original input: `var str = `foo ${"`;\n`${any}"} bar`;`

Reply:
A rough idea is that perhaps we should take a look at the implementation of boa.